fmriprep/workflows/anatomical.py

```diff
@@ -1073,3 +1073,22 @@ def init_anat_derivatives_wf(output_dir, output_spaces, template, freesurfer,
         ])
 
     return workflow
+
+
+def _seg2msks(in_file):
+    """Converts labels to masks"""
+    import nibabel as nb
+    import numpy as np
+    from os import path as op
+
+    nii = nb.load(in_file)
+    labels = nii.get_data()
+
+    out_files = []
+    for i in range(1, 4):
+        ldata = np.zeros_like(labels)
+        ldata[labels == i] = 1
+        out_files.append(op.abspath('label%d.nii.gz' % i))
+        nii.__class__(ldata, nii.affine, nii.header).to_filename(out_files[-1])
+
+    return out_files
```

Comment:
If this splitting is necessary, and we're not using it elsewhere, why not have it internal to the reporting interface? You're already loading all of the ROIs in memory; is there an advantage to writing them to file and then reloading them in a separate interface?

Reply:
We could do without splitting if we leverage the `levels` input of `add_contours` from nilearn. However, for simplicity, I did not provide an input for that to the interface. We can do that in the next iteration - https://github.com/poldracklab/niworkflows/issues/212
I would not invest time on doing the splitting within the interface instead.

Reply:
Yep, with levels we would pass along the segmentation. No extraction of regions needed at all